src/engine/source/base/parseEvent.cpp

```diff
@@ -61,136 +34,34 @@ Event parseOssecEvent(const std::string& event)
 
     const int queue {event[0]};
     parseEvent->setInt(queue, EVENT_QUEUE_ID);
-
-    const bool isFullLocation = (FIRST_FULL_LOCATION_CHAR == event[2]);
-
-    const auto secondColonIdx {event.find(":", 2)};
-
-    // Case: <Queue_ID>:[<Agent_ID>] (<Agent_Name>) <Registered_IP>-><Origin>:<Log>
-    //                  \                                                  /
-    //                   \------------------- LOCATION -------------------/
-    if (isFullLocation)
+    auto locationIdx = event.find(':', LOCATION_OFFSET);
+    if (locationIdx == std::string::npos)
```

Comment:
Wazuh 4.4 includes the new parsing of events, please note: it can be escaped with the `|` character, we must adjust to that.

https://github.com/wazuh/wazuh/blob/2477e9fa50bc1424e834ac8401ce2450a5978e75/src/analysisd/cleanevent.c#L35-L56

Reply:
Implemented